jwst/regtest/regtestdata.py

```diff
@@ -174,6 +176,7 @@ def get_asn(self, path=None, docopy=None):
         # Get each member in the association as well
         with open(self.input) as fp:
             asn = load_asn(fp)
+        self.asn = asn
```

Comment:
This is a good addition.  It may help later on in OKifying assocation inputs if needed.  Thanks!